docs/source/example_use.rst

```diff
@@ -1,3 +1,343 @@
+  worker with review-level ``photographer`` certification.
+
+* username: ``journalism-copy-editor``, password: ``copy-editor``. A worker
+  with ``copy_editor`` certification.
+
+Finally, we've included a management script to start and monitor the workflow.
+Download it to the directory of your project next to ``manage.py`` with::
+
+  wget https://raw.githubusercontent.com/unlimitedlabs/orchestra/stable/example_project/journalism_workflow_ctl.py
+
+Start the workflow
+=====================
+
+To start the workflow, make sure Orchestra is running (``python manage.py
+runserver``), and in another tab, run::
+
+  python journalism_workflow_ctl --new
+
+This will take a bit (because it is automatically running the document
+creation workflow step), but will eventually return a project id (probably
+``1``), which you should store for future use, and output JSON info about
+the project.
+
+Complete the steps
+==================
+
+To navigate the workflow, first log in as the ``journalism-editor`` user and
+request a new task. The interface should look like the image below:
+
+.. image:: ../static/img/journalism_workflow/editor.png
+
+Fill out the high-level story idea and submit the task.
+
+Next, log in as the ``journalism-reporter-1`` worker, and you should now have
+a reporting task available. The interface looks like the image below--use the
+google doc to write your article.
+
+.. image:: ../static/img/journalism_workflow/reporter.png
+
+When you submit, you'll note that the task appears in the 'Awaiting Review'
+section. That's your cue to log in as ``journalism-reporter-2`` and review the
+work. Once you're satisfied with it, accept it.
+
+In parallel to logging in as a reporter, you can log in as
+``journalism-photographer-1`` and ``journalism-photographer-2`` to take and
+review photographs relevant to the article. You should see an interface like the
+image below, which encourages you to add photos to a shared 'Raw Photos' folder.
+The interface should look like the below:
+
+.. image:: ../static/img/journalism_workflow/photographer.png
+
+Once you've accepted the photography as ``journalism-photographer-2``, the
+machine task to auto-process the photos should run. Our implementation simply
+makes any images in 'Raw Photos' greyscale, but you could imagine more
+complicated adjustments.
+
+Finally, log in as ``journalism-copy-editor`` to give the article a headline
+and caption the photos. You should observe that your photos have been greyscaled
+as desired, as in the image below:
+
+.. image:: ../static/img/journalism_workflow/copy_editor.png
+
+Once you submit the task, the workflow is done! You've successfully
+coordinated 6 expert workers and 2 machine tasks to construct a news story.
+
+Verify the final JSON output
+============================
+
+You'll note that our workflow didn't actually lay the article out in its final
+print or electronic form. That's because this workflow was likely run from a
```

Comment:
```
That's because, in reality, this workflow would have been kicked off by a newsroom's content management system with auto-layout capabilities based on the JSON the project produced.
```